templates/CRM/Event/Page/DashBoard.tpl

```diff
@@ -116,14 +113,17 @@
       {/if}
     </tr>
     {/foreach}
-
     </tbody>
     </table>
     {if $eventSummary.total_events GT 10}
      <div><a href="{crmURL p='civicrm/admin/event' q='reset=1'}"><i class="crm-i fa-chevron-right" role="img" aria-hidden="true"></i> {ts}Browse more events{/ts}...</a></div>
     {/if}
+    <div class="crm-clearfix">
+      <div class="pull-right">
```

Comment:
Is this a bootstrap thing? That seems ok just note that it doesn't "pull right" on all civi installs. The links appear on the left.

Reply:
You're right (no pun intended). RiverLea supports `pull-right` and `float-right`, but even the newer convention of Bootstrap is `float-right`, and the old `css/civicrm.css` seems to support that too.

I updated the PR.